src/System.Management.Automation/engine/hostifaces/PowerShell.cs

```diff
@@ -598,6 +599,10 @@ public sealed class PowerShell : IDisposable
         private bool _isBatching = false;
         private bool _stopBatchExecution = false;
 
+        // Delegates for asynchronous invocation/termination of PowerShell commands
+        private Func<IAsyncResult, PSDataCollection<PSObject>> _endInvokeMethod;
+        private Action<IAsyncResult> _endStopMethod;
```

Comment:
Readonly is done locally (will commit after a few more updates). Inline initialization cannot be done -- doing so generates error CS0236: A field initializer cannot reference the non-static field, method, or property 'PowerShell.EndInvoke(IAsyncResult)'. I would much prefer inline initialization but because of this error I cannot, so I initialize them in constructors because that is the only way I know of to avoid error CS0236.

Reply:
> doing so generates error CS0236

Good point. I didn't do it in the code myself and missed that.